frontend/src/lib/mocks/arrow/differentColumnTypes.ts

```diff
@@ -0,0 +1,1238 @@
+/**
+ * @license
+ * Copyright 2018-2021 Streamlit Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// df1 = pd.DataFrame([["baz", "1"], ["qux", "2"]], columns=[1, 2], index=["i1", "i2"])
+// df2 = pd.DataFrame([["foo", "1"], ["bar", "2"]], columns=["c1", "c2"], index=["i1", "i2"])
+
+export const DIFFERENT_COLUMN_TYPES = new Uint8Array([
```

Comment:
Can you add a short docstring describing what this means? ("Raw data for a dataframe that has blah blah blah properties"). Same for the other mock datas.

Reply:
Done